hack/make/ubuntu

```diff
@@ -150,6 +158,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
Do we actually have documentation that explains how to use an LSM effectively in combination with Docker?  Without telling users how to actually use this in the real world, we're just blindly setting it and hoping it helps (and with our default profile, making sure it helps for our own desired "use cases" that we think the majority of people use Docker for/in).